README.md

```diff
@@ -4,67 +4,79 @@
-* **ATAqC**: Annotation-based analysis including TSS enrichment and comparison to Roadmap DNase.
-* **Genomes**: Pre-built database for GRCh38, hg19, mm10, mm9 and additional support for custom genomes.
+* **Portability**: The pipeline run can be performed across different cloud platforms such as Google, AWS and DNAnexus, as well as on cluster engines such as SLURM, SGE and PBS.
+* **User-friendly HTML report**: In addition to the standard outputs, the pipeline generates an HTML report that consists of a tabular representation of quality metrics including alignment/peak statistics and FRiP along with many useful plots (IDR/TSS enrichment). An example of the [HTML report](https://storage.googleapis.com/encode-pipeline-test-samples/encode-atac-seq-pipeline/ENCSR889WQX/example_output/qc.html). The [json file](docs/example_output/v1.1.5/qc.json) used in generating this report.
+* **Supported genomes**: Pipeline needs genome specific data such as aligner indices, chromosome sizes file and blacklist. We provide a genome database downloader/builder for hg38, hg19, mm10, mm9. You can also use this [builder](docs/build_genome_database.md) to build genome database from FASTA for your custom genome.
 
 ## Installation
 
-1) Install [Caper](https://github.com/ENCODE-DCC/caper#installation). Caper is a python wrapper for [Cromwell](https://github.com/broadinstitute/cromwell). Make sure that you have python3(> 3.4.1) installed on your system.
+1) Git clone this repo.
 
-  ```bash
-  $ pip install caper
-  ```
+	```bash
+	$ cd
+	$ git clone https://github.com/ENCODE-DCC/atac-seq-pipeline
+	```
 
-2) Read through [Caper's README](https://github.com/ENCODE-DCC/caper) carefully.
+2) Install [Caper](https://github.com/ENCODE-DCC/caper#installation). Caper is a python wrapper for [Cromwell](https://github.com/broadinstitute/cromwell).
 
-3) Run a pipeline with Caper.
+	> **IMPORTANT**: Make sure that you have python3(> 3.4.1) installed on your system.
 
-## Running pipelines without Caper
+	```bash
+	$ pip install caper  # use pip3 if it doesn't work
+	```
 
-Caper uses the cromwell workflow execution engine to run the workflow on the platform you specify.  While we recommend you use caper, if you want to run cromwell directly without caper you can learn about that [here](docs/deprecated/OLD_METHOD.md).
+3) Read through [Caper's README](https://github.com/ENCODE-DCC/caper) carefully. Find an instruction for your platform. 
+	> **IMPORTANT**: Configure your Caper configuration file `~/.caper/default.conf` correctly for your platform.
 
-## DNAnexus
+## Running a pipeline locally with Caper
 
-You can also run our pipeline on DNAnexus without using Caper or Cromwell. There are two ways to build a workflow on DNAnexus based on our WDL.
+1) Prepare an input JSON file. We will use a subsampled example input JSON based on URLs. Caper will automatically download all fastqs and reference human genome data recursively.
```

Comment:
"The first step in running the pipeline will be to create a correct input JSON file."
@leepc12 : I am not sure what we mean here : "We will use a **subsampled** example input JSON **based on URLs**"

Reply:
May be we can reword it a bit to make it more clear. Looks like we are using a sub-sample of "fastq" files. In other words- it is "custom trimmed fastq file"- right?
It would be useful to explain which files are included in the  ""atac.genome_tsv" : "https://storage.googleapis.com/encode-pipeline-genome-data/hg38_caper.tsv","  and what could one do if they had to use for example another "reference genome file" instead of the one that is specified in this example.

Reply:
thanks and sure i agree. firstly i removed that confusing "subsampled" thing. About the input JSON/output  specification, I will make a separate PR.